src/components/datatable.component.ts

```diff
@@ -452,6 +453,13 @@ export class DatatableComponent implements OnInit, AfterViewInit, DoCheck {
   @Output() rowContextmenu = new EventEmitter<{ event: MouseEvent, row: any }>(false);
 
   /**
+   * The context menu was invoked on a header cell.
+   *
+   * @memberOf DatatableComponent
+   */
+  @Output() headerContextmenu = new EventEmitter<{ event: MouseEvent, name: string, prop: string }>(false);
```

Comment:
What do you think about consolidating row and header into a single event with a type?